sdks/typescript/src/apache_beam/transforms/group_and_combine.ts

```diff
@@ -69,7 +69,7 @@ type Combiner<I> = CombineFn<I, any, any> | ((a: any, b: any) => any);
  */
 export class GroupBy<T, K> extends PTransformClass<
   PCollection<T>,
-  PCollection<KV<K, Iterable<T>>>
+  PCollection<KV<K, T[]>>
```

Comment:
Does `T[]` not satisfy `Iterable<T>`? I don't think we want to promise an array, as we may want to give other iterables in the future. (This is important to support values sets that do not fit into memory.)